app/modules/claim.js

```diff
@@ -117,6 +125,58 @@ const getUpdatedClaimableAmount = async ({
   return pollForUpdatedClaimableAmount({ net, address, claimableAmount })
 }
 
+export const handleN3GasClaim = async ({
+  FROM_ACCOUNT,
+  dispatch,
+  net,
+}: {
+  FROM_ACCOUNT: {},
+  dispatch: DispatchType,
+  net: string,
+}) => {
+  // TODO:
+  // - Ledger support/integration
+
+  let endpoint = await getNode(net)
+  if (!endpoint) {
+    endpoint = await getRPCEndpoint(net)
+  }
+
+  const CONFIG = {
+    account: FROM_ACCOUNT,
+    rpcAddress: endpoint,
+    // TODO: this will have to by dynamic based on test/mainnets
+    networkMagic: 844378958,
```

Comment:
Is this the correct mainnet number?  Just making sure this isn't a missed TODO prior to mainnet usage.